src/ml_gym/frontend/dashboard/src/App.tsx

```diff
@@ -0,0 +1,157 @@
+import { useState, useEffect } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import Template from './template/template';
+import FlowBoard from './routes/flow_board'
+import AnalysisBoard from './routes/analysis_board'
+import Settings from './routes/settings'
+import Throughput from './routes/throughput_board'
+import io from 'socket.io-client';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { useAppDispatch } from "./app/hooks"
+import { IOStatsType, FilterConfigType } from "./app/datatypes"
+import { jobStatusAdded } from "./features/jobsStatus/jobsStatusSlice"
+import { modelStatusAdded } from "./features/modelsStatus/modelsStatusSlice"
+import { modelEvaluationAdded } from "./features/modelEvaluations/modelEvaluationsSlice"
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+
+const socket = io("http://localhost:7000");
+
+
+export default function App() {
+
+  const [sideBarExpanded, setSideBarExpanded] = useState<boolean>(false);
+  const [selectedPageId, setSelectedPageId] = useState<number>(0)
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
  const [selectedPageId, setSelectedPageId] = useState<number>(0);
```